.cargo/audit.toml

```diff
@@ -31,4 +31,12 @@ ignore = [
     # proc-macro-error is unmaintained, but hard to replace right now.
     # Follow https://github.com/Kyuuhachi/syn_derive/issues/4
     "RUSTSEC-2024-0370",
+
+    # The instant package is unmaintained, but hard to replace right now because
+    # parking_lot depends on it.
+    "RUSTSEC-2024-0384",
+
+    # The derivative package is unmainained, but hard to replace right now
+    # because ark-poly depends on it.
```

Comment:
it seems they switched to `educe` 
https://github.com/arkworks-rs/algebra/commit/85d17f579976a62d3398de5c8571150f584b35f6

If we are very lucky upgrading this `ark-poly` might be quick 🤞

Reply:
No need to spend time on this before unblocking CI.